src/Donations/ValueObjects/DonationStatus.php

```diff
@@ -11,22 +11,22 @@
  * @method static DonationStatus COMPLETE()
  * @method static DonationStatus REFUNDED()
  * @method static DonationStatus FAILED()
- * @method static DonationStatus CANCELED()
+ * @method static DonationStatus CANCELLED()
  * @method static DonationStatus ABANDONED()
  * @method static DonationStatus PREAPPROVAL()
  * @method static DonationStatus PROCESSING()
  * @method static DonationStatus REVOKED()
- * @method static DonationStatus RENEWAL()
+ * @method static DonationStatus RENEWAL() @deprecated
```

Comment:
I'm not sure if this actually marks it deprecated but we should add a description as well.

Reply:
Fun fact, I learned there's no way to mark magic methods as deprecated. I even reported this to PhpStorm: https://youtrack.jetbrains.com/issue/WI-68849/Marking-a-magic-method-method-as-deprecated-does-nothing

Reply:
Further details included in 7c1ac47e6